geometry_msgs/Vector3[] cost
geometry_msgs/Vector3[] clearly